app/models/c_rater/feedback_functionality.rb

```diff
@@ -57,7 +57,9 @@ def copy_of_feedback(prev_feedback)
     status: prev_feedback.status,
     answer: self,
     answer_text: self.answer_text,
-    score: prev_feedback.score
+    score: prev_feedback.score,
+    feedback_text: prev_feedback.feedback_text,
+    response_info: prev_feedback.response_info
```

Comment:
It seems like the indentation is off here. 
Also you could use `create!` instead of `new` followed `save!`

Reply:
@scytacki  we followed your advice in 4e886b2